webhook/client.go

```diff
@@ -163,21 +163,21 @@ func ValidatePayloadWithTolerance(payload []byte, header string, secret string,
 }
 
 type ConstructEventOptions struct {
-	// Validates event timestamps using a custom tolerance window. If this is
+	// Validates event timestamps using a custom Tolerance window. If this is
 	// not set and `ignoreTolerance` is false, will default to
```

Comment:
Nit: Fix casing in the references throughout (eg. `ignoreTolerance` -> `IgnoreTolerance`)